include/eve/concept/scalar.hpp

```diff
@@ -110,10 +111,19 @@ concept product_scalar_value = detail::scalar_tuple<T>();
 template<typename T>
 concept arithmetic_scalar_value = plain_scalar_value<T> || product_scalar_value<T>;
 
+//================================================================================================
+//! @concept element_value
+//! @brief Represents any type that can be loaded into a eve::wide
+//! The concept `element_value<T>` is satisfied if and only if T satisfies either
+//! transparent_value or arithmetic_scalar_value.
+//================================================================================================
+ template <typename T>
+ concept element_value = transparent_value<T> || arithmetic_scalar_value<T>;
```

Comment:
1.
I'm not so sure about the name. I don't think it tells me anything.
You will also encounter `arithmetic_value` and then what? 

Let's think about it.

One option is to say that all enums and such are actually arithmentic values.
This is a bit ify, because `std::is_arithmetic` is false for enums.

I would suggest `extended_arithmetic_scalar_value`

Reply:
What about we say, that's a type can be put into a wide, so it is vectorizable.
What about `concetp vectorizable` ?.??